modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/RpcResponseFactory.java

```diff
@@ -73,4 +73,23 @@ default Message newResponse(RaftMessagesFactory msgFactory, int code, String fmt
             eBuilder.errorMsg(String.format(fmt, args));
         return eBuilder.build();
     }
+
+    /**
+     * Creates an error response with parameters.
+     *
+     * @param leaderId New leader id
+     * @param msgFactory Raft message factory
+     * @param code error code with raft info
+     * @param fmt message with format string
+     * @param args arguments referenced by the format specifiers in the format string
+     * @return a response instance
+     */
+    default Message newResponse(String leaderId, RaftMessagesFactory msgFactory, int code, String fmt, Object... args) {
```

Comment:
- Do we have list of possible codes somewhere?
- Is it possible to create errorResponse with 0 //OK code?

Reply:
- Fixed
- Yes
- Will create separate issue about this idea
- Yes

Reply:
Now the line is too long.